neurolib/optimize/exploration/exploration.py

```diff
@@ -54,38 +61,46 @@ def __init__(self, model=None, parameterSpace=None, evalFunction=None):
         # todo: use random ICs for every explored point or rather reuse the ones that are generated at model initialization
         self.useRandomICs = False
 
+        if filename is None:
+            filename = "exploration.hdf"
```

Comment:
```suggestion
        filename = filename or "exploration.hdf"
```

Reply:
This is very ninja, never saw it! Thanks!